schema/google/showcase/v1beta1/echo.proto

```diff
@@ -237,6 +240,13 @@ message EchoResponse {
 
   // The other request ID specified or autopopulated in the request.
   string other_request_id = 4;
+
+
+  message RepeatedValues {
+    repeated string header_values = 1;
+  }
+  // The HTTP header to echo as specified in the request
+  map<string, RepeatedValues> http_request_header_value = 5;
```

Comment:
Why not just echo it back in the response headers directly? That's what we've done in the past...

Reply:
Fixed in https://github.com/googleapis/gapic-showcase/pull/1501/commits/8d1f1031e539275c0be3e0812867e7fcb75a81eb.

See trailing metadata below
```
> responses.trailing_metadata()[0]
_Metadatum(key='x-goog-api-version', value='v1_20240408')
```